execution/contextTree.sh

```diff
@@ -1218,7 +1218,9 @@ function upgrade_cmdb_repo_to_v1_3_0() {
   local -A account_mappings
   readarray -t account_files < <(find "${GENERATION_DATA_DIR}" -type f -name "account.json" )
   for account_file in "${account_files[@]}"; do
-    provider_id="$( jq -r '.Account.ProviderId' <"${account_file}" )"
+    provider_id="$( jq -r '.Account.ProviderId | select(.!=null)' < "${account_file}" )"
+    [[ -e "${provider_id}" ]] && provider_id="$( jq -r '.Account.AWSId | select(.!=null)' < "${account_file}" )"
+    [[ -e "${provider_id}" ]] && provider_id="$( jq -r '.Account.AzureId | select(.!=null)' < "${account_file}" )"
```

Comment:
same as above

Reply:
@ml019 updated the flag with the commit 

313e9e5